app/views/users/edit.html.erb

```diff
@@ -0,0 +1,38 @@
+</h1>Edit an user </>
+<% 
+=begin %>
+ <form action="/users" method="post" accept-charset="UTF-8">
```

Comment:
- [ ] The method here should not be `'post'` as this is for creating a new entry. 

![Screen Shot 2020-11-18 at 6 30 24 PM](https://user-images.githubusercontent.com/48804258/99565599-2114a380-29cc-11eb-84d0-cf446c871b5d.png)